packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -122,10 +139,33 @@ export const FeedContainer = ({
           data-testid="posts-feed"
         >
           {inlineHeader && header}
-          {showSearch && <SearchBar className={{ container: 'mb-8' }} />}
+          {isV1Search && (
+            <FlexRow className="gap-3">
+              <SearchBarInput
+                className={{
+                  container: 'max-w-2xl w-full flex flex-1',
+                  field: 'w-full',
+                }}
+                showProgress={false}
+                onSubmit={onSearch}
+              />
+              {besideSearch}
+            </FlexRow>
+          )}
+          {isV1Search && (
+            <FlexRow className="flex flex-1 mt-4">
```

Comment:
Still not a fan of these 😂
Why don't we simply pass the row, these are super prone to changes down the line and just don't really make sense at this point.

Reply:
Yes correct :) Think we should just remove them to be honest.

Reply:
Amazing. Should be removed by now.